### PR TITLE
Regex modify

### PR DESCRIPTION
**Pull request by lwcislo**

### CODE REVIEW

**Comment by pietrushnic** [comment]
@lwcislo not well tested:

```
[15:07:06] pietrushnic:pce-fw-builder git:(regex_modify) $ ./build.sh dev-build ../coreboot apu2.trust                                                                                     
~/storage/projects/2019/pcengines/coreboot ~/storage/projects/2019/pcengines/pce-fw-builder
ERROR: Tag not recognized 4.10
```

**Comment by pietrushnic** [comment]
@lwcislo @artur-rs maybe we should basic tests here?

File: build.sh [hunk 1]
```diff
@@ -2,22 +2,10 @@
 
 check_if_legacy() {
     case "$1" in
-        4.[1-9][0-9]*)
-            return 0
-            ;;
-        4.[0-3]*)
-            return 1
-            ;;
-        4.[4-9]*)
```
Comment (miczyg1) [inline build.sh]: @lwcislo please fix the missing tags that do not start with `v`

File: build.sh [hunk 2]
```diff
@@ -8,6 +8,21 @@ check_if_legacy() {
...
+        4\.[1-9][0-9]*)
+            return 0
+            ;;
+        4\.[2-7]*)
+            return 1
+            ;;
+        4\.0*)
+            return 1
+            ;;
+        4\.1[^0-9]*)
```
Comment (miczyg1) [inline build.sh]: @lwcislo doesn't it duplicate somehow the rule in the line 11?

**Pull request merged**